src/components/skills-quiz/SearchCourseCard.jsx

```diff
@@ -21,9 +21,13 @@ import { useSelectedSkillsAndJobSkills } from './data/hooks';
 import { useDefaultSearchFilters } from '../search/data/hooks';
 import { UserSubsidyContext } from '../enterprise-user-subsidy';
 
-const getCourseSkills = (course) => (
-  course.skill_names?.length > 0 ? course.skill_names.slice(0, MAX_VISIBLE_SKILLS_CARD) : null
-);
+const getCourseSkills = (course, selectedJobSkills) => {
```

Comment:
What if we name this method `getCommonSelectedAndCourseSkills` ?

Reply:
done